packages/web-reporter-ui/ReporterView.tsx

```diff
@@ -37,9 +40,15 @@ const Report = ({ results }: { results: TestCaseResult[] }) => {
     averageTestCaseResult
   );
 
+  const saveResultsToZIP = () => {
+    exportRawDataToZIP(results);
+  };
+
   return (
     <>
-      <ReportSummary results={averagedResults} />
+      <Header saveToZIPCallBack={saveResultsToZIP} />
```

Comment:
realizing that maybe we can avoid having the header if we have `results.length === 1` what do you think?